recipes/open62541/all/conanfile.py

```diff
@@ -0,0 +1,300 @@
+        "enable_pubsub_deltaframes": [True, False],                             # Enable sending of delta frames with only the changes
+        "enable_pubsub_informationmodel": [True, False],                        # Enable PubSub information model twin
+        "enable_pubsub_informationmodel_methods": [True, False],                # Enable PubSub informationmodel methods
+        "enable_pubsub_custom_publish_handling": [True, False],                 # Use a custom implementation for the publish callback handling
+        "enable_json_encoding": [True, False],                                  # Enable Json encoding (EXPERIMENTAL)
+        "enable_statuscode_descriptions": [True, False],                        # Enable conversion of StatusCode to human-readable error message
+        "enable_typedescription": [True, False],                                # Add the type and member names to the UA_DataType structure
+        "enable_nodeset_compiler_descriptions": [True, False],                  # Set node description attribute for nodeset compiler generated nodes
+        "enable_malloc_singleton": [True, False],                               # Use a global variable pointer for malloc (and free, ...) that can be switched at runtime
+        "msvc_force_static_crt": [True, False],                                 # Force linking with the static C-runtime library when compiling to static library with MSVC
+        "enable_discovery_semaphore": [True, False],                            # Enable Discovery Semaphore support
+        "debug_dump_pkgs": [True, False],                                       # Dump every package received by the server as hexdump format
+        "enable_hardening": [True, False],                                      # Enable Hardening measures (e.g. Stack-Protectors and Fortify)
+        "enable_subscriptions_alarms_conditions": [True, False],                # Enable the use of A&C. (EXPERIMENTAL)
+        "enable_parsing": [True, False],                                        # Utility functions that require parsing (e.g. NodeId expressions)
+        "force_cpp": [True, False],                                             # Force compilation with a C++ compiler
+        "enable_da": [True, False],                                             # Enable OPC UA DataAccess (Part 8) definitions
+        "enable_pubsub_eth_uadp_xdp": [True, False],                            # Enable Subscribe UADP over Ethernet using XDP
+        "enable_pubsub_eth_uadp_etf": [True, False],                            # Use ETF implementation for the ETH_UADP publish
+        "enable_pubsub_mqtt": [True, False],                                    # Enable publish/subscribe with mqtt (experimental)
+        "namespace_zero": ["minimal", "reduced", "full"]                        # Completeness of the generated namespace zero (minimal/reduced/full)
```

Comment:
I'd love to see options grouped together:
 * move `enable_pubsub_eth_uadp_xdp`, `enable_pubsub_eth_uadp_etf `, `enable_pubsub_mqtt` together with the others related to `enable_pubsub...`
 * `enable_discovery_semaphore`...

Reply:
It makes sense to group `enable_pubsub_eth_uadp , enable_pubsub_eth_uadp_xdp, enable_pubsub_eth_uadp_etf` under one option as an enumeration, since, as I understand, these options are mutually exclusive